packages/@react-spectrum/datepicker/test/DateRangePicker.test.js

```diff
@@ -1020,7 +1022,7 @@ describe('DateRangePicker', function () {
       expect(endField).not.toHaveAttribute('aria-describedby');
 
       let description = group.getAttribute('aria-describedby').split(' ').map(d => document.getElementById(d).textContent).join(' ');
-      expect(description).toBe('Selected Range: February 3, 2020 at 8:00 AM to February 10, 2020 at 10:00 AM');
+      expect(description).toBe('Selected Range: February 3, 2020, 8:00 AM to February 10, 2020, 10:00 AM');
```

Comment:
When I recreate this test in Stroybook it behaves like it used to with the "at" instead of a comma.